electrum/gui/qt/paytoedit.py

```diff
@@ -82,7 +85,30 @@ def parse_address_and_amount(self, line):
         amount = self.parse_amount(y)
         return TxOutput(out_type, out, amount)
 
+    def parse_opencap(self, x):
+        username, domain = opencap.validateAlias(x)
+        if username == "" or domain == "":
+            return x
+
+        host, dnssec = opencap.getHost(domain)
+        if host == "":
+            return x
+
+        if not dnssec:
+            # maybe warn here
+            pass
+
+        response = requests.get(
+            "https://" + host + "/v1/addresses?alias="+username+"$"+domain)
+        json_data = json.loads(response.text)
+        for addressObject in json_data:
+            if addressObject["address_type"] == 100 or 101:
```

Comment:
The [documentation is inconsistent](https://github.com/opencap/protocol/tree/5a8d42b4848c84413c4f1a745979b885a9377dbb/AddressTypes) but shouldn't 102 be also allowed?